### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by kimaldis** [Pyroh/SwiftMark#6]
Downloaded the project, tried to build. 76 compiler errors. Xcode 8, what am I missing?

**Comment by l65l** [comment]
I guess you tried to compile swiftMarkiOS target. The OSX sample compiles. I can see 54 errors for iOS. I check what is going on.

**Comment by l65l** [comment]
It is a build setting issue. you can quick fix buy setting - Use Legacy Swift language Version - to NO on the SwiftMarkiOS target.

**Comment by kimaldis** [comment]
You're right, I was building for iOS. Sorry, I'm a bit green still on all things Xcode but I should have spotted that.
Thanks.

**Comment by kimaldis** [comment]
I don't know how much you care about dealing with html code in markdwon, I know it's a fuzzy topic, but if found that:

     <p>
        # some title
     </p>

markdwon within the tag didn't get parsed. However;

   <p>
   
   # some title
   </p>

the markdown did get parsed correctly.

**Comment by l65l** [comment]
hello @kimaldis , this looks like another issue. Please open a new issue with more details like input string and output string in both cases. Thanks
